### ISSUE TITLE
Please update the app.

### ISSUE THREAD

**Issue by sohate** [lhaeger/AutoRaise#10]
Please update app for latest version https://github.com/sbmpost/AutoRaise

**Comment by zenminimalist** [comment]
Yes, I second that. And that would make an even more awesome person 🙂

**Comment by sbmpost** [comment]
@zenminimalist
@sohate 

I have prepared an update but I am not able to make a merge request ATM because of #13.

For the time being, a copy of the updated repo can be found under https://github.com/sbmpost/AutoRaise-UI

**Comment by sohate** [comment]
> @zenminimalist @sohate
> 
> I have prepared an update but I am not able to make a merge request ATM because of #13.
> 
> For the time being, a copy of the updated repo can be found under https://github.com/sbmpost/AutoRaise-UI

The greatest thing of Ihaeger version of this app - it's already compiled. I don't have an XCode, so no chance for me to compile your version of the app. I'm just a noob who do "download .app file and copy it to the 'application' folder". Can you compile it for everyone? Thanks!

**Comment by sohate** [comment]
@sbmpost

> @zenminimalist @sohate
> 
> I have prepared an update but I am not able to make a merge request ATM because of #13.
> 
> For the time being, a copy of the updated repo can be found under https://github.com/sbmpost/AutoRaise-UI

The greatest thing of Ihaeger version of this app - it's already compiled. I don't have an XCode, so no chance for me to compile your version of the app. I'm just a noob who do "download .app file and copy it to the 'application' folder". Can you compile it for everyone? Thanks!

**Comment by sbmpost** [comment]
@sohate 
The problem with providing binaries is that they will not just run because of security related restrictions imposed by the OS. So if manual compilation isn't an option, we will have to wait for @lhaeger to check on #13

**Comment by sbmpost** [comment]
@sohate
@zenminimalist 

So I actually managed to produce a signed binary version with GUI.
See: https://github.com/sbmpost/AutoRaise/blob/master/AutoRaise.dmg

Please let me know if this works for you.

**Comment by sohate** [comment]
@zenminimalist 

Got this every time:

Error 503 Backend is unhealthy

Backend is unhealthy

Error 54113

Details: cache-tyo11951-TYO 1659385192 3434762126

Varnish cache server

**Comment by sohate** [comment]
@zenminimalist 

It doesn't focus Safari fullscreen video. Works with IINA fullscreen, but not with Safari's fullscreen. Can you fix it?

**Comment by sbmpost** [comment]
@zenminimalist 

> It doesn't focus Safari fullscreen video. Works with IINA fullscreen, but not with Safari's fullscreen. Can you fix it?

Can you create a separate issue for this via https://github.com/sbmpost ?

P.S: you might want to drag the delay slider on tick higher as a workaround.

**Comment by sohate** [comment]
@zenminimalist 

Already did, doesn't work. I'll create a new issue https://github.com/sbmpost right now.
